media/js/firefox/share-page.es6.js

```diff
@@ -24,4 +24,19 @@ function copyText() {
         textCopiedMessage.style.display = 'none';
         copyButtonText.style.display = 'block';
     }, 2500);
+
+    // UA
+    window.dataLayer.push({
+        event: 'in-page-interaction',
+        eAction: 'button click',
+        eLabel: 'Copy and share'
+    });
+
+    // GA4
+    window.dataLayer.push({
+        event: 'widget_action',
+        type: 'copy',
+        action: 'click',
```

Comment:
I'd like to move away from ever typing "click" as a value in GA4. I think `copy` is more appropriate here and it's fine that it's a repeat of type.

That said, it's not a blocker.

Reply:
I changed both actions to "copy"